### PR TITLE
Enwiki accuracy

### PR DESCRIPTION
**Pull request by halfak**
Adds analysis and demonstration of high AUC enwiki revert classifier.

This also fixes minor issues in scripts for gathering features/train_test and adds a new "score" script to try classifications on real data.

### CODE REVIEW

File: R/util.R [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+				xval
+			}
+		},
+		x,
+		then
+	)
+	
+}
+
+clean.monthly_creations = function(dt){
```
Comment (he7d3r) [inline R/util.R]: Is this supposed to be here?

File: ores/new_model.py [hunk 2]
```diff
@@ -24,46 +21,13 @@
 def main():
     args = docopt.docopt(__doc__)
     
-    Model = import_from_path(args['<class>'])
-    features = tuple(import_from_path(c) for c in args['<feature>'])
-    if args['--kwargs'] is not None:
-        kwargs = yaml.load(open(args['--kwargs']))
```
Comment (he7d3r) [inline ores/new_model.py]: It seems the `import yaml` is not needed anymore

File: ipython/Enwiki Revert Classifier.ipynb [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+      "\n",
+      "A few of the revisions won't be found -- which is fine.  We'll just have to work with the remaining data.  So, let's load it in!"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "feature_scores = []\n",
+      "with open(\"../datasets/enwiki.features_reverted.5k.tsv\") as f:\n",
```
Comment (he7d3r) [inline ipython/Enwiki Revert Classifier.ipynb]: This assumes the current directory is `ores/ores`, right?

File: ipython/Enwiki Revert Classifier.ipynb [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+      "## Testing the models\n",
+      "\n",
+      "First, the linear kernel."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "import matplotlib.pyplot as plt\n",
```
Comment (he7d3r) [inline ipython/Enwiki Revert Classifier.ipynb]: At first, I got `ImportError: No module named 'matplotlib'`. Then I executed `pip install matplotlib` in the terminal but an error occurred: http://dpaste.com/35AM0JK.

File: ipython/Enwiki Revert Classifier.ipynb [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+      "\n",
+      "\n",
+      "plt.figure()\n",
+      "plt.plot(stats['roc']['fpr'],\n",
+      "         stats['roc']['tpr'], \n",
+      "         label='ROC curve')\n",
+      "plt.plot([0, 1], [0, 1], 'k--')\n",
+      "plt.xlabel('False Positive Rate')\n",
+      "plt.ylabel('True Positive Rate')\n",
+      "plt.show()"
```
Comment (he7d3r) [inline ipython/Enwiki Revert Classifier.ipynb]: I wasn't able to see the image until I used this: http://stackoverflow.com/a/15900439/2062663.

File: ores/new_model.py (hunk 2 above)
Comment (halfak) [inline ores/new_model.py]: Yes.  Good point

File: ipython/Enwiki Revert Classifier.ipynb (hunk 3 above)
Comment (halfak) [inline ipython/Enwiki Revert Classifier.ipynb]: Assumes that the current directory is ore/ipython which is required by the way that "ipython notebook" works.

File: ipython/Enwiki Revert Classifier.ipynb (hunk 4 above)
Comment (halfak) [inline ipython/Enwiki Revert Classifier.ipynb]: Yeah.  Matplotlib sucks, but it's what we have.  :(

File: ipython/Enwiki Revert Classifier.ipynb (hunk 5 above)
Comment (halfak) [inline ipython/Enwiki Revert Classifier.ipynb]: The figure should appear if you run it via ipython notebook or the viewer.  E.g. http://nbviewer.ipython.org/github/halfak/Objective-Revision-Evaluation-Service/blob/enwiki_accuracy/ipython/Enwiki%20Revert%20Classifier.ipynb

File: ipython/Enwiki Revert Classifier.ipynb [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+     "prompt_number": 6
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "And there you have it. "
+     ]
+    },
+    {
```
Comment (he7d3r) [inline ipython/Enwiki Revert Classifier.ipynb]: Empty cell at the end of the page.

Comment (halfak) [inline ipython/Enwiki Revert Classifier.ipynb]: It seems like this is something that ipython notebook does when you ask it execute the last block.  If you feel strongly, I'll remove it, but it seems to me that we should let this kind of issue pass.

**Pull request merged**